src/rust/storage/seg/src/ttl_buckets/tests.rs

```diff
@@ -0,0 +1,9 @@
+use super::*;
+use rustcommon_time::CoarseDuration;
+
+#[test]
+fn bucket_index() {
+    let ttl_buckets = TtlBuckets::new();
+    assert_eq!(ttl_buckets.get_bucket_index(CoarseDuration::ZERO), 1023);
```

Comment:
a bit wimpy? Could use a few intermediate values as examples (one for each range, for example)

Reply:
Updated the test to cover the boundaries for each bucket in the full range.

Reply:
awesome. Above and beyond :)